packages/std/src/testing/mock.rs

```diff
@@ -269,6 +255,26 @@ impl Api for MockApi {
 }
 
 impl MockApi {
+    /// Returns [MockApi] with Bech32 prefix set to provided value.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use cosmwasm_std::Addr;
+    /// # use cosmwasm_std::testing::MockApi;
+    /// #
+    /// let mock_api = MockApi::with_prefix("juno");
+    /// let addr = mock_api.addr_make("creator").to_string();
+    ///
+    /// assert_eq!("juno1h34lmpywh4upnjdg90cjf4j70aee6z8qqfspugamjp42e4q28kqsksmtyp", addr);
+    /// ```
+    pub fn with_prefix(prefix: &'static str) -> Self {
+        Self {
+            bech32_prefix: prefix,
+            ..Default::default()
+        }
+    }
```

Comment:
We could make this composable:

```rust
let mock_api = MockApi::default().with_prefix("juno");
let addr = mock_api.addr_make("creator").to_string();
```

by taking owned Self as the first argument.

Reply:
Done.